### DISCUSSION TITLE
TFT guage example - TFT_Meter_5 reposition

### DISCUSSION THREAD

**Discussion by Aclarky1969** [Bodmer/TFT_eSPI#1464] [General]
Hi

I'm currently using a ESP32 TTGO and was wondering if its possible to easily
reposition the guage on the screen. I need to do this with my application. 

Thanks for the great examples for the TTGO.

Alan

**Comment by Bodmer** [comment]
Yes, but it was not written with that in mind. Therefore you will need to look at adding offsets in the graphics  plotting coordinates, or re-write with a position parameter.
